addons/html_editor/static/tests/html_field.test.js

```diff
@@ -1531,3 +1532,57 @@ describe("sandbox", () => {
         );
     });
 });
+
+describe("direction config", () => {
+    test("ltr direction", async () => {
+        onRpc("partner", "web_save", ({ args }) => {
+            expect(args[1].txt).toBe("<div></div>");
+            expect.step("web_save");
+        });
+        defineParams({
+            lang_parameters: {
+                direction: "ltr",
+            },
+        });
+        await mountView({
+            type: "form",
+            resId: 1,
+            resModel: "partner",
+            arch: `
+            <form>
+                <field name="txt" widget="html"/>
+            </form>`,
+        });
+        expect(".odoo-editor-editable").toHaveAttribute("dir", "ltr");
+        const node = queryOne(".odoo-editor-editable p");
+        setSelection({ anchorNode: node, anchorOffset: 0, focusNode: node, focusOffset: 1 });
+        switchDirection(htmlEditor);
```

Comment:
use a real user flow. Open command palette and select switch direction (insertText(htmlEditor, "/switch") + press("enter")